packages/rpc-provider/src/mock/index.ts

```diff
@@ -38,6 +38,7 @@ const l = logger('api-mock');
  * @return {ProviderInterface} The mock provider
  */
 export default class Mock implements ProviderInterface {
+  public body: any;
```

Comment:
I really don't think there should be a body here. Body is used to mock the http response

Reply:
Ah, that was a leftover from an attempt to fix another linting error. Consider it removed.